checks.d/mysql.py

```diff
@@ -1,30 +1,33 @@
 # stdlib
-import os
 import sys
 import re
 import traceback
+from contextlib import closing, contextmanager
+from collections import defaultdict
 
 # 3p
 import pymysql
+try:
+    import psutil # permissions issues?
```

Comment:
nothing - will be removed. `psutil` has like two different levels of usage stats - not all stats are available if you don't have privileges, but we should handle that case just fine.